backend/benefit/applications/services/ahjo_payload.py

```diff
@@ -11,13 +11,19 @@
 
 MANNER_OF_RECEIPT = "sähköinen asiointi"
 
+def prepare_title(application: Application) -> str:
```

Comment:
This commit can be ignored, truncating the title conditionally is not needed.

Reply:
I forgot that I had a previous commit and did not amend the actual solution commit.